src/styles/blocks/content/card.scss

```
@@ -0,0 +1,86 @@
+.card-container {
+  &__card {
+    margin: 0 24px 46px;
+    max-width: 200px;
+  }
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 16px auto 40px;
+  }
+
+  &__product-name {
+    margin-bottom: 4px;
+    font-weight: 500;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+    @extend %secondary-card-text-styles;
+    color: $secondary-text-color;
+  }
+
+  &__rate {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__comments {
+    align-self: center;
+    @extend %secondary-card-text-styles;
+  }
+
+  &__cost {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__cost-text {
+    color: $secondary-text-color;
+  }
+
+  &__cost-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 112%;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 10px 0;
+    min-height: 40px;
+    width: 100%;
+
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 114%;
+    text-transform: uppercase;
+    text-align: center;
+    text-decoration: none;
+    color: #fff;
+
+    background-color: $interactive-color;
+    border: 1px solid $interactive-color;
```

Add default transparent border.
```suggestion
    border: 1px solid transparent;
```